charts/common/templates/_deployment.yaml

```diff
@@ -293,6 +293,13 @@ spec:
             - name: CONSORTIUM_ENABLED
               value: {{ .Values.consortiumEnabled | quote }}
           {{- end }}
+          {{- if .Values.rtrEnabled }}
+            - name: LEGACY_TOKEN_TENANTS
+              value: ''
+          {{- else -}}
```

Comment:
This not gonna be working, please use below approach:
- name: LEGACY_TOKEN_TENANTS
   value: {{- if .Values.rtrEnabled }} {{- else }} '*' {{- end }}